releases/release-5.2.0.md

```diff
@@ -210,7 +210,7 @@ Support running the `tiup playground` command on Mac computers with Apple M1 chi
     - Support completing the garbage collection automatically for the bindings in the "deleted" status [#26206](https://github.com/pingcap/tidb/pull/26206)
     - Support showing whether a binding is used for query optimization in the result of `EXPLAIN VERBOSE` [#26930](https://github.com/pingcap/tidb/pull/26930)
     - Add a new status variation `last_plan_binding_update_time` to view the timestamp corresponding to the binding cache in the current TiDB instance [#26340](https://github.com/pingcap/tidb/pull/26340)
-    - Support reporting an error when starting binding evolution or running `admin evolve bindings` to ban the baseline evolution (currently disabled in the on-premises TiDB version because it is an experimental feature) affecting other features [#26333](https://github.com/pingcap/tidb/pull/26333)
+    - Support reporting an error when starting binding evolution or running `admin evolve bindings` to ban the baseline evolution (currently disabled in the TiDB Self-Hosted because it is an experimental feature) affecting other features [#26333](https://github.com/pingcap/tidb/pull/26333)
```

Comment:
```suggestion
    - Support reporting an error when starting binding evolution or running `admin evolve bindings` to ban the baseline evolution (currently disabled in TiDB Self-Hosted because it is an experimental feature) affecting other features [#26333](https://github.com/pingcap/tidb/pull/26333)
```